dream_layer_frontend/src/components/Slider.tsx

```
@@ -41,38 +45,171 @@ const Slider = ({
+  // Get tooltip content if tooltipKey is provided
+  const tooltipContent = tooltipKey ? getTooltipContent(tooltipKey as any) : null;
+
   return (
-    <div className="mb-4">
-      <div className="flex items-center justify-between">
+    <div className="mb-4 p-3 rounded-lg border border-border/50 bg-card/30 glass-morphism hover:shadow-sm transition-all duration-300">
+      <div className="flex items-center justify-between mb-3">
         {label && (
-          <div className="text-sm font-medium">
+          <div className="text-sm font-medium flex items-center">
             <span dangerouslySetInnerHTML={{ __html: label }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)